recipes/hdmedians/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "hdmedians" %}
+{% set version = "0.13" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 230f80e064d905c49a1941af1b7e806e2f22b3c9a90ad5c21fd17d72636ea277
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cython >=0.23
+    - numpy
+    - pip
+    - python
+    - nose
+  run:
+    - cython >=0.23
+    - numpy
+    - python
+    - nose
```

Comment:
cython is needed at runtime, as well as buildtime? And nose is needed for building and running, not just for testing?